mscore/prefsdialog.cpp

```diff
@@ -434,26 +425,18 @@ void PreferenceDialog::start()
                                                       }
                                                 },
                                           [&]() { // update function
-                                                language->blockSignals(true);
-                                                for (int i = 0; i < language->count(); ++i) {
-                                                      if (language->itemText(i).startsWith(preferences.getString(PREF_UI_APP_LANGUAGE))) {
-                                                            language->setCurrentIndex(i);
-                                                            break;
-                                                            }
-                                                      }
-                                                language->blockSignals(false);
-                                                // ???
                                                 language->blockSignals(true);
                                                 language->clear();
                                                 QString lang = preferences.getString(PREF_UI_APP_LANGUAGE);
+                                                cout << lang.toStdString() << endl;
```

Comment:
if so it should rather be `qDebug()`

Reply:
About that, I was actually wondering why a few days ago. I mean, std is included anyway, so why not use its convenient `cout` instead of this clunky C style `qDebug`?

Reply:
Actually, std is not included in every file. 

(and this change has been removed in the latest commit)